Configurations.md

```diff
@@ -1705,13 +1705,52 @@ pub enum Foo {}
 pub enum Foo {}
 ```
 
-## `merge_imports`
+## `imports_merge_style`
+
+Merge together related imports based on their paths.
+
+This option requires `reorder_imports`, which is enabled by default.
+
+- **Default value**: `Preserve`
+- **Possible values**: `Preserve`, `Crate`, `Module`
+- **Stable**: No (tracking issue: [#3362](https://github.com/rust-lang/rustfmt/issues/3362))
 
-Merge multiple imports into a single nested import.
+#### `Preserve` (default):
+
```

Comment:
I think it'd probably be useful to include some additional text for the variants here that described their behavior in natural language. For example, what about something like this for Preserve:

```suggestion
Do not perform any merging and preserve the original structure written by the developer
```